Runtime/Scripts/MoveToCenterOfFeatureCollection.cs

```diff
@@ -64,21 +64,21 @@ private Vector3 PullbackToFitBounds(Transform targetTransform, Bounds bounds)
             return position;
         }
 
-        private Bounds CalculateCenterAndExtents(FeatureCollection featureCollection) 
+        private Bounds CalculateCenterAndExtents(FeatureCollection featureCollection)
         {
             double[] boundingBox = featureCollection.BoundingBoxes ?? featureCollection.DerivedBoundingBoxes();
             int epsgId = featureCollection.EPSGId();
 
             var realWorldTopLeft = new Coordinate(epsgId, boundingBox[0], boundingBox[1], elevation);
             var realWorldBottomRight = new Coordinate(epsgId, boundingBox[2], boundingBox[3], elevation);
-            var topLeft = CoordinateConverter.ConvertTo(realWorldTopLeft, CoordinateSystem.Unity);
-            var bottomRight = CoordinateConverter.ConvertTo(realWorldBottomRight, CoordinateSystem.Unity);
-            var width = bottomRight.Points[0] - topLeft.Points[0];
-            var depth = bottomRight.Points[2] - topLeft.Points[2];
+            Coordinate topLeft = new Coordinate(realWorldTopLeft.ToUnity());
```

Comment:
realWorldTopLeft is al een Coordinate, dus dan hoef je niet nog een nieuwe coordinate te maken